### PR TITLE
Selenium: fix existed selenium tests from 'factory' package according to factory creation changes 

### PR DESCRIPTION
**Pull request by SkorikSergey**
### What does this PR do?
This PR adapts selenium tests from ```factory``` package according to factory creation changes(https://github.com/eclipse/che/pull/12232).

Selenium tests to fix:
- DirectUrlFactoryWithKeepDirectoryTest
- DirectUrlFactoryWithSpecificBranchTest
- DirectUrlFactoryWithRootFolderTest

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/12295

### CODE REVIEW

**Comment by SkorikSergey** [comment]
ci-test

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithSpecificBranchTest.java [hunk 1]
```diff
@@ -81,59 +64,44 @@ public void setUp() throws Exception {
...
-    events.waitExpectedMessage(
-        String.format(
-            "Project: %s | cloned from: %s | remote branch: refs/remotes/origin/%s | local branch: %s",
-            repositoryName, repositoryName, SECOND_BRANCH_NAME, SECOND_BRANCH_NAME),
-        UPDATING_PROJECT_TIMEOUT_SEC);
-
-    projectExplorer.waitAndSelectItem(repositoryName);
-    pullRequestPanel.waitOpenPanel();
-
-    projectExplorer.expandPathInProjectExplorer(repositoryName + "/my-lib");
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithSpecificBranchTest.java:123]: If there is a reason stop checking content and type of cloned project?

Comment (SkorikSergey) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithSpecificBranchTest.java:123]: Theia doesn't have project types.

Comment (dmytro-ndp) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithSpecificBranchTest.java:123]: I see.
What about checking content of cloned project which is specific to branch?

Comment (SkorikSergey) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithSpecificBranchTest.java:123]: I will add content checking to this test.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithRootFolderTest.java [hunk 2]
```diff
@@ -13,41 +13,31 @@
...
-import org.openqa.selenium.TimeoutException;
+import org.eclipse.che.selenium.pageobject.theia.TheiaIde;
+import org.eclipse.che.selenium.pageobject.theia.TheiaProjectTree;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 /** @author Musienko Maxim */
-@Test(groups = {GITHUB, FLAKY})
+@Test(groups = {GITHUB, OPENSHIFT, FLAKY})
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithRootFolderTest.java]: FLAKY group is redundant here so as we got rid of random failure.

**Review by dmytro-ndp — APPROVED** [review]

**Pull request merged**